src/Id/Handler.php

```diff
@@ -1,7 +1,9 @@
 <?php
 namespace Sesshin\Id;
 
-use Sesshin\League\EntropyGenerator;
+use Sesshin\EntropyGenerator\EntropyGeneratorInterface;
+use Sesshin\EntropyGenerator\Uniq;
+use Sesshin\Exception;
```

Comment:
Sure, look [here](https://github.com/sobstel/sesshin/blob/master/src/Id/Handler.php#L72). In the current form, it's not clear whats used there, since we're in a namespace. You should either use `\Exception` or `\Sesshin\Exception`. I choosed `\Sesshin\Exception`.